flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/SpillableSubpartitionTest.java

```diff
@@ -319,7 +319,7 @@ public void testConsumeSpillablePartitionSpilledDuringConsume() throws Exception
 		assertEquals(2, partition.getBuffersInBacklog());
 		assertEquals(partition.getBuffersInBacklog(), read.buffersInBacklog());
 		read.buffer().recycleBuffer();
-		assertEquals(2, listener.getNumNotifications());
+		assertEquals(1, listener.getNumNotifications());
```

Comment:
maybe add a small comment that we don't get a new notification (anymore!) since the buffer is already available at the `getNextBuffer()` call?